test/passing/tests/js_source.ml.ref

```diff
@@ -532,7 +531,6 @@ let n2 = extension_name (Bar 1)
 let t = extension_id (Bar 2) = extension_id (Bar 3) (* true *)
 
 let f = extension_id (Bar 2) = extension_id Foo (* false *)
-
```

Comment:
This is surprising indeed, no idea what is causing this

Reply:
There are still many instances of this bug. This should be fixed before the next release to avoid having a new intermediate bug between the previous state and the ideal state.

Repro (`profile=conventional`):
```ocaml
let x = foo 1 (* x *)
let y = bar (1 + 1) (* y *)

let z = baz (* z *)
```